pyro/contrib/gp/models/sgpr.py

```diff
@@ -135,24 +135,28 @@ def model(self):
         W = Kuf.trtrs(Luu, upper=False)[0]
```

Comment:
Would be cleaner to make this `W` agree with the eventual upstream `LowRankMultivariateNormal` and define
```py
W = Kuf.trtrs(Luu, upper=False[0].t()

D = noise.expand(W.shape[0])
...
Wffdiag = W.pow(2).sum(dim=1)
...
f_var = D + W.pow(2).sum(dim=1)
...
dist.LowRankMultivariateNormal(f_loc, W, D)
```
?

Reply:
Yup! I'll change things in autoguide too. :)